### PR TITLE
Add `plot_optimization_history(error_bar=True)`

### PR DESCRIPTION
**Pull request by toshihikoyanase**
## Motivation

This is a follow-up PR for #3.
See https://github.com/optuna/optuna-visualization-regression-tests/pull/3#issuecomment-1175924431.

## Changes

- Add `plot_optimization_history(error_bar=True)` page.
- I added kwargs of plot functions to filename and URL. I'm not sure the generated filenames and URLs are safe since they can include `=` and `,`.

image

image

### CODE REVIEW

**Comment by toshihikoyanase** [comment]
@HideakiImamura I added plots with `error_bar=True` based on your suggestion. Could you check the changes, please?

**Review by HideakiImamura — COMMENTED** [review]
Thanks for the PR. Basically, LGTM. It seems that the generated filename is not correct. Could you fix it?

File: visual_regression_tests.py [hunk 1]
```diff
@@ -98,107 +107,137 @@ def generate_plot_files(
 
 
 def generate_optimization_history_plots(
-    studies: List[Tuple[str, StudiesType]], base_dir: str
+    studies: List[Tuple[str, StudiesType]],
+    base_dir: str,
+    plot_kwargs: Dict[str, Any],
 ) -> List[Tuple[str, str, str]]:
+    filename_prefix = "history"
+    if len(plot_kwargs) == 0:
```
[the remark this answers is not in the document]
Comment (toshihikoyanase) [reply visual_regression_tests.py]: You're absolutely right. I'll update the rest of the plot functions.

**Review by HideakiImamura — APPROVED** [review]
LGTM!

**Pull request merged**